python/lsst/dax/imgserv/getimage/imagegetter_v1.py

```diff
@@ -533,11 +512,12 @@ def _cutout_from_src(self, src_image, xy_center_x, xy_center_y, width,
         else:
             self._log.debug("cutout image wanted is OUTSIDE source image -> None")
             return None
-        if isinstance(src_image, afw_image.ExposureF):
+        if isinstance(src_image, afw_image.Exposure):
             self._log.debug("co_box pix_ulx={} pix_end_x={} pix_uly={} pix_end_y={}"
                   .format(pix_ulx, pix_ulx + width, pix_uly, pix_uly + height))
             # img will keep wcs from source image
-            image = afw_image.ExposureF(src_image, co_box)
+            # image = afw_image.ExposureF(src_image, co_box)
+            image = self._image_from_butler(data_id, co_box)
```

Comment:
But this seems wrong.  You're retrieving the image again, when it looks like it's already available in `src_image`.  You should either move this to the initial retrieval of `src_image` or else use `afw` methods to subset the image (as was done previously).

Reply:
I'm aware of this, and would like to optimize the subimage fetching process.  Ideally in a single butler.get().  Question:  can I make the first call a 'lazy' fetch that does not load the actual image data, but only the headers, so I can use the header info to compute the cutout dimension?